Assets/Scripts/ControlNodeObject.cs.orig

```diff
@@ -0,0 +1,131 @@
+﻿using System.Collections;
```

Comment:
remove this file. `.orig` files are artifacts of merge conflicts and should never be committed.